pkg/kubelet/prober/results/results_manager.go

```diff
@@ -58,6 +58,17 @@ func (r Result) String() string {
 	}
 }
 
+func (r Result) ToPrometheusType() float64 {
+	switch r {
+	case Success:
+		return 1
+	case Failure:
+		return 0
```

Comment:
nit: slight preference to make success 0 and failure 1 so it aligns with process exit codes.

Reply:
Done.